flink-runtime/src/main/java/org/apache/flink/runtime/resourcemanager/ResourceManager.java

```diff
@@ -108,6 +111,8 @@
 
 	public static final String RESOURCE_MANAGER_NAME = "resourcemanager";
 
+	protected final Logger log = LoggerFactory.getLogger(getClass());
```

Comment:
Unnecessary. The logger is already defined in `RpcEndpoint`, which is a super class of `ResourceManager.

Reply:
Done.